packages/grafana-data/src/types/datasource.ts

```diff
@@ -320,6 +320,7 @@ abstract class DataSourceApi<
 
   /**
    * Used in explore
+   * @deprecated instead implement `DataSourceToggleableQueryFiltersSupport`
```

Comment:
I would deprecate this after `DataSourceToggleableQueryFiltersSupport` is released. Also `modifyQuery` is used in Table visualisation for Prometheus queries and there is no immediate plan to change this in Table, so maybe we should deprecate after Table viz has this implementeded as well.

Reply:
No longer deprecated.